frontend/src/data/tasks.json

```diff
@@ -8,7 +8,7 @@
       {
         "answerKey": "yes-income",
         "descriptionEn": "If you earn money while collecting your public pensions, your income may be higher than you expect and you could be taxed more. You may need to set aside or withhold more money for taxes. If you do not end up working or earning money while collecting benefits, your income might be lower than you planned.",
-        "descriptionFr": "Si vous gagnez de l'argent, tout en percevant vos pensions publiques, votre revenu peut être plus élevé que prévu et vous pourriez être davantage imposé. Vous devrez peut-être mettre de côté ou conserver plus d'argent pour les impôts. Si vous ne travaillez pas ou ne gagnez pas d'argent pendant que vous recevez des prestations, votre revenu peut être inférieur à ce que vous aviez prévu.",
+        "descriptionFr": "Si vous gagnez de l’argent, tout en recevant vos pensions publiques, votre revenu peut être plus élevé que prévu et vous pourriez être davantage imposé. Vous devrez peut-être mettre de côté ou conserver plus d’argent pour les impôts. Si vous ne travaillez pas ou ne gagnez pas d’argent pendant que vous recevez des prestations, votre revenu peut être inférieur à ce que vous aviez prévu.",
```

Comment:
`'` apostrophe should be used instead of `’` right single quotation mark. This comment is also for the other places.